docs/runtimes/application.ipynb

```diff
@@ -82,16 +82,306 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    @app.route('/')
</pre><p>can remove hello, docs, and secret endponits they are not related</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>